docs/Config_Reference.md

```diff
@@ -1262,6 +1264,10 @@ See also: [extended g-code commands](G-Codes.md#z_thermal_adjust).
 #max_z_adjustment:
 #   Maximum absolute adjustment that can be applied to the Z axis [mm]. The
 #   default is 99999999.0 mm (unlimited).
+#sensor:
```

Comment:
Just a suggestion: It should be possible to reuse any existing sensor by:
`sensor_type: temperature_combined`

So, it may not be necessary to reimplement same functionality here.

Reply:
Yes, that does work with config like this:

```
[z_thermal_adjust nozzle]
temp_coeff: -0.00046459
sensor_type: temperature_combined
sensor_list: extruder
combination_method: max
min_temp: 0
max_temp: 400
maximum_deviation: 1
max_z_adjustment: 0.5
```

I can remove the `sensor` option. I think this idea of a `sensor_list` is great for the frame expansion as well.

Reply:
Updated to remove the `sensor` stuff. That makes the PR really short now. 😄